xskillscore/core/np_deterministic.py

```diff
@@ -219,3 +284,107 @@ def _mae(a, b, weights, axis):
         )
     else:
         return absolute_error.mean(axis=axis)
+
+
+def _mad(a, b, weights, axis):
```

Comment:
Since this is the median (and there's no sum or mean funcs) this shouldn't take a `weights` argument.